.remarkrc.js

```diff
@@ -6,4 +6,8 @@ exports.settings = {
   incrementListMarker: false
 };
 
-exports.plugins = ['remark-preset-lint-travi', [require('remark-toc'), {tight: true}]];
+exports.plugins = [
+  'remark-preset-lint-travi',
+  [require('remark-toc'), {tight: true}],
```

Comment:
worth mentioning that i hope to define additional presets for these details at some point, but havent taken the time to figure out the right way to capture the details above